superset/models/core.py

```diff
@@ -415,7 +415,11 @@ def get_df(  # pylint: disable=too-many-locals
         mutator: Optional[Callable[[pd.DataFrame], None]] = None,
     ) -> pd.DataFrame:
         sqls = self.db_engine_spec.parse_sql(sql)
-        engine = self.get_sqla_engine(schema)
+
+        engine = self.get_sqla_engine(schema=schema, user_name=username)
```

Comment:
`user_name` is deprecated in `get_sqla_engine`.

Reply:
I have updated the call to `get_sqla_engine`

Reply:
@john-bodley  @nytai  How can we get this to the next level our colleague was an intern whose internship has ended and we would like to get this merged into main superset. Apologies for my ignorance on what steps are pending on our part if any.  But as none of us are approved reviewers we need a bit of guidance on how to get this into main.